docs/source/deployment/security_checklist/security_checklist_template.md

```diff
@@ -0,0 +1,137 @@
+# Security checklist
+
+Running on SHM/SREs deployed using commit xxxxxx
+
+## Summary
+
+- :white_check_mark: x tests passed
+- :partly_sunny: x tests partially passed (see below for more details)
+- :fast_forward: x tests skipped (see below for more details)
+- :x: x tests failed (see below for more details)
+
+## Details
+
+- Any additional details as referred to in the summary
+
+### Multifactor Authentication and Password strength
+
+- :white_check_mark:/:partly_sunny:/:fast_forward:/:x: Check: Users can reset their own password
```

Comment:
I think I forgot to include these options.

Do you want to change the target of this PR to the release branch? I will handle the conflicts.